tests/periph_spi_dma/Makefile

```diff
@@ -0,0 +1,3 @@
+FEATURES_REQUIRED += periph_dma
+# Include everything else from the periph_spi test
+include ../periph_spi/Makefile
```

Comment:
You want to change that `Makefile` to also do

```Makefile
FEATURES_REQUIRED += periph_spi
```
currently it will do
```Makefile
FEATURES_REQUIRED = periph_spi
```
and thus disable the DMA.

Reply:
Yeah…

Fixed